### PR TITLE
qcoro 0.4.0

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **qcoro/0.4.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by prince-chrismc — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

**Review by SSE4 — APPROVED** [review]

File: recipes/qcoro/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def build_requirements(self):
+        self.build_requires("cmake/3.21.3")
+
+    def requirements(self):
+        self.requires("qt/6.2.2")
+        self.requires("expat/2.4.2", override=True)
```
Comment (SpaceIm) [inline recipes/qcoro/all/conanfile.py:62]: Why expat version is overridden?

Comment (ericLemanissier) [reply recipes/qcoro/all/conanfile.py:62]: I don't remember. probably a conflict in qt requirements ?

Comment (SpaceIm) [reply recipes/qcoro/all/conanfile.py:62]: Would be nice to remove it in the next PR.

**Pull request merged**